spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/CreateTableEncKey.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical
+
+import org.apache.spark.sql.catalyst.expressions.Attribute
+
+case class CreateTableEncKey(
```

Comment:
Nit / non-blocking: In the Spark repo, are these normally termed with `Command` at the end (like `CreateTableCommand`, etc)?

Reply:
Right, these normally termed with `Command` at the end in Spark repo, but seems all the logical commands in `spark3-extensions` under `org.apache.spark.sql.catalyst.plans.logical` don't have `Command` in the end, e.g. `SetIdentifierFields`, so I guess I will just be consistent with others.